environments/hub.json

```diff
@@ -95,6 +95,23 @@
     "toIdType": "MSISDN",
     "toIdValue": "27713803912",
     "toIdValueDelete": "27713803913",
+    
+    "toAccentIdType": "MSISDN",
+    "toAccentIdValue": "97039819999",
+    "toAccentIdDOB": "2000-01-01",
+    "toAccentIdFirstName": "Seán",
+    "toAccentIdMiddleName": "François",
+    "toAccentIdLastName": "Nuñez",
+    "toAccentIdFspId": "payeefsp",
+
+    "toBurmeseIdType": "MSISDN",
+    "toBurmeseIdValue": "2224448888",
+    "toBurmeseIdDOB": "1990-01-01",
+    "toBurmeseIdFirstName": "ကောင်းထက်စံ",
+    "toBurmeseIdMiddleName": "အောင်",
+    "toBurmeseIdLastName": "ဒေါ်သန္တာထွန်",
+    "toBurmeseIdFspId": "payeefsp",
+
```

Comment:
Please note this file is just for reference. We need to include these new environment variables in the mojaloop values file for the helm release.

Reply:
sure, thanks Vijay